vllm/v1/struct_output/request.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import dataclasses
+import functools
+import json
+from concurrent.futures import Future
+from concurrent.futures._base import TimeoutError
+from typing import Optional, Union, cast
+
+from vllm.sampling_params import SamplingParams
+from vllm.v1.struct_output.grammar import (Grammar, StructuredOutputKey,
+                                           StructuredOutputOptions)
+
+
+@dataclasses.dataclass
+class StructuredOutputRequest:
+
+    sampling_params: SamplingParams
+    _grammar: Optional[Union[Future[Grammar],
+                             Grammar]] = dataclasses.field(default=None)
```

Comment:
Just curious: why not just use `None`?
```suggestion
    _grammar: Optional[Union[Future[Grammar], Grammar]] = None
```